python/lsst/ts/rotator/rotator_csc.py

```diff
@@ -244,11 +246,10 @@ def telemetry_callback(self, server):
         server : `lsst.ts.hexrotcomm.CommandTelemetryServer`
             TCP/IP server.
         """
-        # TODO DM-26384: use the new timestamp in the telemetry from
-        # the low-level controller, instead of this time.
-        # Also consider computing actual velocity using delta position
-        # and that (presumably accurate) timestamp.
-        curr_tai = salobj.current_tai()
+        tel_time = astropy.time.Time(
+            server.header.mjd, server.header.mjd_frac, format="mjd", scale="utc"
+        )
+        tel_tai_unix = salobj.tai_from_utc(tel_time)
```

Comment:
What about issuing a warning event when timestamps from low level differs significantly from local clock? This might prevent some time-synchronization problems.. Shall I start a JIRA ticket for this or you will be able to add this check now? I am not sure if there is any warning event in ts_rotator XML available for that..

Reply:
Or maybe even better, rejecting low-level updates when timestamps differ..

Reply:
I worry it will require too much code for little gain. It requires configuring a max tolerance, then dealing with the fact that the data arrives at 10-20 Hz and I certainly don't want to print warnings at that rate, so I need logic for "I have already reported this", and a way of resetting that after N "in spec" times. If you feel strongly about it feel free to file a ticket. A Watcher alarm is another option (though I'd rather not have it responding to an event at that rate if I can avoid it, which suggests polling...)